router.go

```diff
@@ -660,7 +668,7 @@ func (r *Router) Find(method, path string, c Context) {
 				// act similarly to any node - consider all remaining search as match
 				i = l
 			} else {
-				for ; i < l && search[i] != '/' && search[i] != '-'; i++ {
+				for ; i < l && search[i] != '/' && !(search[i] == '-' || search[i] == '.'); i++ {
```

Comment:
@aldas This is the best I could come up with, after spending some time trying around some stuff. Turns out that the dependency on "-" or "." is removed from the "route adding" and instead it is now on the "route detection". But one place should definitely have it or else, it cannot distinguish the params.

Reply:
actually I might be wrong here. Let me go over this again